scripts/environment_setup.sh

```diff
@@ -15,11 +15,11 @@ sudo apt-get update && sudo apt-get install -y awscli docker-ce docker-ce-cli co
 sudo curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.8/bin/linux/amd64/kubectl
 sudo chmod +x /usr/bin/kubectl
 
-echo -n ${K8S_CLUSTER_CERT} | base64 -d > ./ca.crt
-kubectl config set-cluster ${K8S_CLUSTER_NAME} --certificate-authority=./ca.crt --server=https://api.${K8S_CLUSTER_NAME}
-kubectl config set-credentials ${SERVICE_ACCOUNT} --token=${K8S_TOKEN}
-kubectl config set-context ${K8S_CLUSTER_NAME} --cluster=${K8S_CLUSTER_NAME} --user=${SERVICE_ACCOUNT} --namespace=${K8S_NAMESPACE}
-kubectl config use-context ${K8S_CLUSTER_NAME}
+echo -n ${K8S_CLUSTER_LIVE1_CERT} | base64 -d > ./ca.crt
+kubectl config set-cluster ${K8S_CLUSTER_LIVE1_NAME} --certificate-authority=./ca.crt --server=https://api.${K8S_CLUSTER_LIVE1_NAME}
+kubectl config set-credentials ${SERVICE_ACCOUNT} --token=${K8S_LIVE1_TOKEN}
+kubectl config set-context ${K8S_CLUSTER_LIVE1_NAME} --cluster=${K8S_CLUSTER_LIVE1_NAME} --user=${SERVICE_ACCOUNT} --namespace=${K8S_NAMESPACE}
+kubectl config use-context ${K8S_CLUSTER_LIVE1_NAME}
```

Comment:
If this is for the `live` cluster should these be `K8S_CLUSTER_LIVE_NAME` etc instead of `LIVE1`?

Reply:
That makes perfect sense... unless some numpty just created all the secrets with the wrong name
![image](https://user-images.githubusercontent.com/6757677/140943025-3da74efc-7b5b-4337-83eb-77d45c33ff5d.png)
🤦‍♂️

Reply:
Also, I cannot remember the encoded/cleartext setup of those values so it's a coin toss whether either will work anyway!